workflow/scripts/prioritization/filtering.py

```
@@ -52,17 +53,30 @@ def assign_scores(epitope_seq, scores):
             seq.to_csv(tmpsfile, sep="\t", header=False, index=False)
 
             # run immunogenicity
-            result = subprocess.run(
-                    ['python', 
-                     'workflow/scripts/immunogenicity/predict_immunogenicity.py',
-                     tmpsfile.name],
-                    stdout = subprocess.PIPE,
-                    universal_newlines = True
-            )
+            try:
+                result = subprocess.run(
```

Ruff (0.15.13)

[error] `subprocess` call: check for execution of untrusted input

(S603)